content/docs/rendering-elements.md

```diff
@@ -1,75 +1,74 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Me-render Elemen
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Elemen adalah blok pembangun terkecil dalam aplikasi React
```

Comment:
I feel like removing pembangun still makes sense. Also don't forget full stop at the end